api/environment.go

```diff
@@ -102,6 +116,20 @@ func (e *Environment) AddVirtualMachine(client SkytapClient, vmId string) (*Envi
 	return e, errors.New("Unable to determine source of VM, no environment or template url found")
 }
 
```

Comment:
In go, its typical practice to keep functions with struct receivers in the same file as the struct. It may make sense to rewrite this with a signature similar to vm.AddNetworkInterface.